index.js

```diff
@@ -289,8 +290,8 @@ function choosePluralForm(text, locale, count) {
 // interpolation placeholder.
 var dollarRegex = /\$/g;
 var dollarBillsYall = '$$';
-var tokenRegex = /%\{(.*?)\}/g;
-function interpolate(phrase, options) {
+function interpolate(phrase, options, tokenPattern) {
+  var tokenRegex = new RegExp(tokenPattern, 'g');
```

Comment:
I think it would be a very bad idea to allow people to pass arbitrary regex data in.

I think that if we want this to be customizable, we'd need a way to *parse* the input, and manually construct the regex safely.